docs/client/connecting-kn-to-your-cluster.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Connecting the `kn` Client to your cluster"
+linkTitle: "Connecting to your cluster"
+weight: 10
+type: "docs"
+---
+
+The `kn` Client requires a `kubeconfig` file to connect to your cluster.
```

Comment:
Are there any requirements that need to be listed here? 

Its not clear what the task is here? Does the user need to take action?

Reply:
It's something that's a bigger Kubernetes concept, I've added a link for more details below: https://github.com/knative/docs/pull/2047/files/f896f7269dc7e3fd59e11f1c9050a3f853234786#diff-ad04a060c15f7df4940a8878b45d1a63R14

AFAIK it's something they should have by default?
I'm not sure if we need to include more than that? @rhuss can you advise if there's some specific action we need to call out to make this clearer?

Reply:
I think this needs to be revised for clarity. ie "requires" sounds like an action item for the user (and there is nothing here that is requesting an action). if this is just the conceptual overview (and not a task as the title indicates ie "connecting to the...") then the title and filename should be changed too. 

Example for this particular line: "the kn client uses the kubeconfif file of your cluster to establish a connection when .....(missing detail: say why kn connects here)"

On the other hand, the user does need to do something to that kubeconfig file in order to connect, we need to state that here. For example, is there a particular element in the kubeconfig file that is "required", if yes, then we should state to look for that ..."ensure the...".

Thanks for all the help with fleshing out the missing info @abrennan89 !
I'm being temporarily assigned to another project so i'll let @samodell take it from here.

Reply:
@rhuss re Richie's question, **why** does `kn` need to connect to a cluster?